src/bci_build/package/valkey.py

```diff
@@ -39,8 +39,8 @@
             )
         ],
         license="BSD-3-Clause",
-        package_list=["valkey"],
-        entrypoint=["/usr/bin/valkey-server"],
+        package_list=["valkey", "sed"],
+        entrypoint=["/usr/bin/valkey-server", "/etc/valkey/valkey.conf"],
```

Comment:
Wouldn't it make sense to keep entrypoint as it is and add `"/etc/valkey/valkey.conf"` as `CMD`?